web/installer

```diff
@@ -52,10 +52,12 @@ function process($argv)
     }
 
     if ($ok || $force) {
-        installComposer($version, $installDir, $filename, $quiet, $disableTls, $cafile, $channel);
-        showWarnings($warnings);
-        showSecurityWarning($disableTls);
-        exit(0);
+        $installer = new Installer($quiet, $disableTls, $cafile);
+        if ($installer->run($version, $installDir, $filename, $channel)) {
```

Comment:
warnings do not prevent running the installation. And they are displayed after the installation so that they are the last output, to stay visible at the end instead of being hidden in the middle of other installation messages.